internal/weldr/upload.go

```diff
@@ -363,6 +375,10 @@ func uploadRequestToTarget(u uploadRequest, imageType distro.ImageType) *target.
 
 			TlsVerify: options.TlsVerify,
 		}
+	case *pulpOSTreeUploadSettings:
```

Comment:
I'm wondering if this new target should be covered also in `targetsToUploadResponses()` function? :thinking: